npm/metrics/testing/test-main.go

```diff
@@ -0,0 +1,60 @@
+package main
```

Comment:
Can you move test-main.go (rename it to main_test.go) out of testing folder.

Reply:
@neaggarwMS a main package has to be in a unique folder, and this file doesn't really test anything. It's more for displaying the metrics you've created on a localhost (outside the realm of kubernetes). I'll give it a more descriptive name